src/pages/Login/Login.js

```diff
@@ -306,11 +306,11 @@ const WebauthnSignupLogin = ({
 								{
 									prfRetryAccepted
 										? (
-											<p className="dark:text-white pb-3">Please interact with your authenticator...</p>
+											<p className="dark:text-white pb-3">{t('registerPasskey.messageInteract')}</p>
 										)
 										: (
 											<>
-												<h3 className="text-2xl mt-4 mb-2 font-bold text-primary dark:text-primary-light">Almost done!</h3>
+												<h3 className="text-2xl mt-4 mb-2 font-bold text-primary dark:text-primary-light">{t('registerPasskey.messageDone')}</h3>
 												<p className="dark:text-white pb-3">
 													{isLogin
 														? 'To finish unlocking the wallet, please authenticate with your passkey once more.'
```

Comment:
Shall we extract these too?

Reply:
```
"registerPasskey": {
	"authOnceMore": "To finish setting up, please authenticate with it once more.",
	"authOnceMoreLogin": "To finish unlocking the wallet, please authenticate with your passkey once more.",
	"authOnceMoreSignup": "To finish setting up your wallet, please authenticate with your passkey once more.",
```

i add both, hmm maybe could we merge the first with the third ?

Reply:
Yes, sounds like a good idea!

Reply:
`To finish setting up` or `To finish setting up your wallet` ? probably the first

Reply:
I was thinking to keep `authOnceMoreSignup` and delete `authOnceMore`, but either way works.

Reply:
Oh, by the way - should `authOnceMoreLogin` be under the `registerPasskey` namespace?

Reply:
probably not, need to move back to loginSignup